README.md

```diff
@@ -318,9 +322,11 @@ The expected command payload is :
 
  ```
  {
-    "cable": 32,
+    "cable": 32.0,
     "ready": true,
-    "consumption": 10
+    "consumption_l1": 10.0,
```

Comment:
Same as previously

Reply:
Because in the code, the update of the value is done only if the consumption is type float